### ISSUE TITLE
Missing classes in 1.2.5 sources

### ISSUE THREAD

**Issue by alice-mkh** [simo415/spc#3]
The 1.2.5 code uses the following classes:
com.sijobe.spc.hook.Player
com.sijobe.spc.util.DynamicClassLoader
com.sijobe.spc.command.CommandHandler
But they are absent both in the sources and the 3.2.2 binaries. So where can I get them? :)

**Comment by q3hardcore** [comment]
There is no way to acquire them. SPC for 1.2.5 was being re-written, however it wasn't completed.
You will need to use 'EntityPlayerSP.java' from here: http://bit.ly/JYaaFZ
(find it under src\minecraft\net\minecraft\src)

**Comment by alice-mkh** [comment]
OK, it seems that it works with these lines commented...

**Comment by alice-mkh** [comment]
Yes, it does. :) https://dl.dropbox.com/u/36782044/spc-1.4.6.png

**Comment by q3hardcore** [comment]
You do these mods, right? http://www.minecraftforum.net/topic/534818-146-olddaysnbxlite-ssp/

Also, there _was_ a good reason SPC was re-written from ground up.
Many of the commands may appear to work, but will not actually have any effect.
This is because they are done client-side, but the world isn't updated serverside.
If you need any help, just ask.

EDIT: actually I guess your mod may workaround this issue.

**Comment by alice-mkh** [comment]
I restored the whole singleplayer in the SSP mod, so this is not an issue. The updated SPC depends on it.
The whole reason I decided to do this is because the official updated SPC was incompatible with SSP.

For now the only issue I see is that the default commands are disabled. IDK if it's better to reimplement them as SPC commands, remove corresponding SPC commands and make them work together or make SPC commands work as the default ones.
In SMP, no matter if local or remote, SPC is just disabled and default commands are used instead.

**Comment by q3hardcore** [comment]
Yeah.. I wasn't exactly sure what you were doing.. well that's cool.

It's pretty easy though just to get the old SPC working in local SMP but with the commands only clientside.
I have done this before, and realized it was completely pointless.

**Comment by alice-mkh** [comment]
I'm sure that I'm not the last person to run into this issue. To prevent creating similar issues here, add about commenting the lines that cause compiling errors to the readme, please. :)

The updated version works great btw. With the "wrapper" command class, SPC commands are even integrated to the vanilla /help and autocompletion. Duplicate commands (vanilla /weather and SPC /weather2 (I renamed it so it doesn't override the vanilla one) etc) are a bit confusing though, but for now I'll leave them like that.

For local SMP — I hope that it will not conflict with the SPC 4._, since in this case the 4._ commands will be available in SMP, and 3.\* ones — in SSP.
